webpack compliance

While trying to use this otherwise great module with webpack, I get a `Cannot find module 'bluebird'` error. 
Looks as if the browserified version basically loads well in webpack, but it avoids webpack's static require substitution. :(

--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,3 @@
 node_modules
 coverage
+browser/bluebird-retry.js

--- a/package.json
+++ b/package.json
@@ -6,7 +6,6 @@
   "scripts": {
     "test": "gulp test"
   },
-  "browser": "./browser/bluebird-retry.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/demmer/bluebird-retry.git"

